newIDE/app/src/EventsSheet/EventsTree/helpers.js

```diff
@@ -34,6 +34,11 @@ export const moveNodeAsSubEvent = ({
     initialEventsList: node.eventsList,
     toIndex: 0,
   });
+  const previousRowIndex = node.nodePath[node.nodePath.length - 1];
```

Comment:
you can probably create a function that does this and reuse it?
something like `getRowIndexFromMove(node, targetNode)`

Reply:
It is a bit tricky since it depends on wether it is moved above or below the target node.
I will make a function to extract the rowIndex though